src/main/java/org/sagebionetworks/web/client/widget/entity/file/ProjectTitleBar.java

```diff
@@ -0,0 +1,66 @@
+package org.sagebionetworks.web.client.widget.entity.file;
+
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+import org.sagebionetworks.repo.model.EntityHeader;
+import org.sagebionetworks.repo.model.Project;
+import org.sagebionetworks.repo.model.entitybundle.v2.EntityBundle;
+import org.sagebionetworks.web.client.EntityTypeUtils;
+import org.sagebionetworks.web.client.security.AuthenticationController;
+import org.sagebionetworks.web.client.widget.SynapseWidgetPresenter;
+import org.sagebionetworks.web.client.widget.entity.FavoriteWidget;
+
+public class ProjectTitleBar implements SynapseWidgetPresenter {
```

Comment:
`ProjectTitleBar` is the previous implementation of `BasicTitleBar`, with no logical changes--just a rename.